rfhub2/cli/rfhub_importer.py

```diff
@@ -106,18 +104,21 @@ def add_collections(self, collections: List[Collection]) -> List[Dict[str, int]]
         loaded_collections = []
         for collection in collections:
             coll_req = self.client.add_collection(collection)
+            if 'unauthorized' in coll_req.get('detail', '').lower():
+                print(coll_req['detail'])
+                exit(1)
```

Comment:
Maybe logging and raising StopIteration error would be cleaner solution than closing the process. Also, maybe checking for status code would be easier?

Reply:
Yeah, status code should be the way, but let me ask You before:
Should I change methods with add/delete collection/keyword so that it resturns tuple of json and return code, or should I append return code to the json itself? First one seems like proper way of doing it, and the latter looks like workaround,, to adapt to current solution.
Having in mind that this will be possibly deleted and moved to some healthcheck and authorizecheck:
>With Basic auth there isn't really a point, but if we would introduce different auth scheme (like LDAP), then the step for logging in should be added and then basic auth could also implement this for the sake of having unified logic. Actually adding this endpoint would be a good way of faiiling fast when user does not pass credentials and would not require handling further in the code. Let's add it, but as a next PR, leave it like this for now, ok?

What should I do?

Reply:
I've added returning of return code along with json in `_post_request` method. Should I add that to `_get_request` and `_delete_request` as well?

Reply:
As I wrote in the other comment, let's leave the status code returned for post like you did it and let's create a task for adding the approach described there. We need to wrap up this PR and start refactoring step by step.